test/e2e/specs/specs-playwright/wp-calypso__coblocks-spec.js

```diff
@@ -0,0 +1,69 @@
+import {
+	setupHooks,
+	DataHelper,
+	LoginFlow,
+	NewPostFlow,
+	GutenbergEditorPage,
+	PublishedPostPage,
+	PricingTableBlock,
+	DynamicHRBlock,
+	HeroBlock,
+} from '@automattic/calypso-e2e';
+
+describe( DataHelper.createSuiteTitle( 'Gutenberg: CoBlocks' ), () => {
+	let gutenbergEditorPage;
+	let pricingTableBlock;
+	let page;
+
+	setupHooks( ( args ) => {
+		page = args.page;
+	} );
+
+	it( 'Log in', async function () {
+		const loginFlow = new LoginFlow( page, 'gutenbergSimpleSiteUser' );
+		await loginFlow.logIn();
+	} );
+
+	it( 'Start new post', async function () {
+		const newPostFlow = new NewPostFlow( page );
+		await newPostFlow.newPostFromNavbar();
+		gutenbergEditorPage = await GutenbergEditorPage.Expect( page );
+	} );
+
+	it( 'Enter post title', async function () {
+		await gutenbergEditorPage.enterTitle( DataHelper.randomPhrase() );
+	} );
+
+	it( 'Insert Pricing Table block and enter price to left table', async function () {
+		const blockHandle = await gutenbergEditorPage.addBlock( 'Pricing Table' );
+		pricingTableBlock = new PricingTableBlock( blockHandle );
+		const price = 888;
+		await pricingTableBlock.enterPrice( 'left', price );
+	} );
+
+	it( 'Insert Dynamic HR block', async function () {
+		await gutenbergEditorPage.addBlock( 'Dynamic HR' );
+	} );
+
+	it( 'Insert Hero block', async function () {
+		const blockHandle = await gutenbergEditorPage.addBlock( 'Hero' );
+		const heroBlock = new HeroBlock( blockHandle );
+		await heroBlock.enterHeading( 'Hero heading' );
+	} );
+
+	it( 'Publish and visit post', async function () {
+		await gutenbergEditorPage.publish( { visit: true } );
+		await PublishedPostPage.Expect( page );
+	} );
+
+	it.each`
+		block                  | name
+		${ PricingTableBlock } | ${ 'Pricing Table' }
+		${ DynamicHRBlock }    | ${ 'Dynamic HR' }
+		${ HeroBlock }         | ${ 'Hero' }
```

Comment:
IMO block name should also be a static property of the block class. This way we create a single source of truth and only one place that will need to be updated in case anything changes.

Reply:
Done!